python-package/setup.py

```diff
@@ -285,6 +291,7 @@ def initialize_options(self) -> None:
         self.mpi = False
         self.hdfs = False
         self.precompile = False
+        self.print_timecosts = False
```

Comment:
```suggestion
        self.time_costs = False
```

Reply:
My apologies. 🤦

Reply:
Haha no worries. Thank you for your contribution!